parachains/runtimes/collectives/collectives-polkadot/src/fellowship/mod.rs

```diff
@@ -108,35 +109,93 @@ morph_types! {
 impl pallet_ranked_collective::Config<FellowshipCollectiveInstance> for Runtime {
 	type WeightInfo = weights::pallet_ranked_collective::WeightInfo<Runtime>;
 	type RuntimeEvent = RuntimeEvent;
-	// Promotion is by any of:
-	// - Root can promote arbitrarily.
+	// Promotions and the induction of new members are serviced by `FellowshipCore` pallet instance.
+	type PromoteOrigin = EnsureNever<pallet_ranked_collective::Rank>;
+	// Demotion is by any of:
+	// - Root can demote arbitrarily.
 	// - the FellowshipAdmin origin (i.e. token holder referendum);
-	// - a vote by the rank *above* the new rank.
-	type PromoteOrigin = EitherOf<
+	// - a vote by the rank two above the current rank.
+	type DemoteOrigin = EitherOf<
 		frame_system::EnsureRootWithSuccess<Self::AccountId, ConstU16<65535>>,
 		EitherOf<
 			MapSuccess<
 				EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>,
 				Replace<ConstU16<9>>,
 			>,
-			TryMapSuccess<EnsureFellowship, CheckedReduceBy<ConstU16<1>>>,
+			TryMapSuccess<EnsureFellowship, CheckedReduceBy<ConstU16<2>>>,
 		>,
 	>;
-	// Demotion is by any of:
-	// - Root can demote arbitrarily.
+	type Polls = FellowshipReferenda;
+	type MinRankOfClass = sp_runtime::traits::Identity;
+	type VoteWeight = pallet_ranked_collective::Geometric;
+}
+
+pub type FellowshipCoreInstance = pallet_core_fellowship::Instance1;
```

Comment:
You could also create a `fellowship_instances` module and name the instance `PolkadotFellowship`, if you expect more to come.  
Same for the salary. Otherwise the name is just the pallet name.